tests/test_subtractTask.py

```diff
@@ -724,6 +724,278 @@ def _compare_apCorrMaps(self, a, b):
+        #  when the template variance plane is incorrect
+        template.variance.array /= scaleFactor
+        science.variance.array /= scaleFactor
+        _run_and_check_images(science, template, sources, statsCtrl,
+                              doDecorrelation=True, doScaleVariance=True, scaleFactor=scaleFactor)
+        _run_and_check_images(science, template, sources, statsCtrl,
+                              doDecorrelation=True, doScaleVariance=False, scaleFactor=scaleFactor)
+        _run_and_check_images(science, template, sources, statsCtrl,
+                              doDecorrelation=False, doScaleVariance=True, scaleFactor=scaleFactor)
+        _run_and_check_images(science, template, sources, statsCtrl,
+                              doDecorrelation=False, doScaleVariance=False, scaleFactor=scaleFactor)
+
+    def test_exposure_properties(self):
+        """Check that all necessary exposure metadata is included
+        with the Score image.
+        """
+        noiseLevel = 1.
+        science, sources = makeTestImage(psfSize=3.0, noiseLevel=noiseLevel, noiseSeed=6)
+        psf = science.psf
+        psfAvgPos = psf.getAveragePosition()
+        psfSize = getPsfFwhm(science.psf)
+        psfImg = psf.computeKernelImage(psfAvgPos)
+        template, _ = makeTestImage(psfSize=2.0, noiseLevel=noiseLevel, noiseSeed=7,
+                                    templateBorderSize=20, doApplyCalibration=True)
+
+        config = subtractImages.AlardLuptonPreconvolveSubtractTask.ConfigClass()
+
+        def _run_and_check_images(doDecorrelation):
+            """Check that the metadata is correct with or without decorrelation.
+            """
+            config.doDecorrelation = doDecorrelation
+            task = subtractImages.AlardLuptonPreconvolveSubtractTask(config=config)
+            output = task.run(template.clone(), science.clone(), sources)
+            psfOut = output.scoreExposure.psf
+            psfAvgPos = psfOut.getAveragePosition()
+            if doDecorrelation:
+                # Decorrelation requires recalculating the PSF,
+                #  so it will not be the same as the input
+                psfOutSize = getPsfFwhm(science.psf)
+                self.assertFloatsAlmostEqual(psfSize, psfOutSize)
+            else:
+                psfOutImg = psfOut.computeKernelImage(psfAvgPos)
+                self.assertImagesAlmostEqual(psfImg, psfOutImg)
+
+            # check PSF, WCS, bbox, filterLabel, photoCalib
+            self.assertWcsAlmostEqualOverBBox(science.wcs, output.scoreExposure.wcs, science.getBBox())
+            self.assertEqual(science.filter, output.scoreExposure.filter)
+            self.assertEqual(science.photoCalib, output.scoreExposure.photoCalib)
+        _run_and_check_images(doDecorrelation=True)
+        _run_and_check_images(doDecorrelation=False)
+
+
+def _computePSFNoiseEquivalentArea(psf):
+    """Compute the noise equivalent area for an image psf
```

Comment:
I'm not familiar with "noise equivalent area" - should this be something available for PSFs in general, or is it really only a thing for these tests? Is it worth a sentence describing what it means here?